src/main/java/org/kuali/kfs/sys/batch/BatchFile.java

```diff
@@ -31,7 +30,6 @@
 
 public class BatchFile extends TransientBusinessObjectBase {
```

Comment:
Now that we're upgrading to a Redis-supported KFS release, I think the only other customization remaining in this overlay is one for fixing an issue related to the older-style batch file lookup (which may no longer be available in KFS). We may want a separate user story to look into potentially removing this overlay, given the changes made to the newer-style lookups since the time of the customization.

Reply:
KFSPTS-26175 has been created